test/glific_web/schema/session_template_test.exs

```diff
@@ -289,39 +289,6 @@ defmodule GlificWeb.Schema.SessionTemplateTest do
     assert message =~ "has already been taken"
   end
 
```

Comment:
instead of deleting the entire testcase, update existing testcase with new changes and how the function behaves now